rsocket/ResumeManager.h

```diff
@@ -15,9 +15,42 @@ class IOBuf;
 
 namespace rsocket {
 
+// Struct to hold information relevant per stream.
+struct StreamResumeInfo {
+  StreamResumeInfo() = delete;
+  StreamResumeInfo(
+      StreamType _streamType,
```

Comment:
Nit: Don't name variables `_foo`, all underscore-prefixed identifiers are "reserved by the system" in C++.